Automattic/wp-calypso
# Add checklist navigation item to the sidebar

#### Changes proposed in this Pull Request

This PR adds a "Customer home" item to Calypso sidebar, linking to the checklist. It would finally give users a way to return to the checklist after getting lost in Calypso.

This is labeled "Customer Home" and not "Checklist" since the new customer home feature is going to replace this link in a couple of weeks and we want to make the transition a bit less confusing.

#### Testing instructions

Go to a site you are owner or admin of, the sidebar should contain the "Customer Home" link:
<img width="277" alt="Zrzut ekranu 2019-08-5 o 19 09 04" src="https://user-images.githubusercontent.com/205419/62482199-d8c07a80-b7b4-11e9-8fb2-079b1eb8bbf3.png">

Navigate to Customer Home, confirm the menu item is active:
<img width="273" alt="Zrzut ekranu 2019-08-5 o 19 09 00" src="https://user-images.githubusercontent.com/205419/62482198-d827e400-b7b4-11e9-8e9c-3659d963df82.png">

As a regular user (not a8c), go to a site you are not owner or admin of, confirm there is no "Customer Home" sidebar item:
<img width="269" alt="Zrzut ekranu 2019-08-5 o 18 43 32" src="https://user-images.githubusercontent.com/205419/62480742-f4298680-b7b0-11e9-993f-20bb5f36e018.png">
